src/commands/config.js

```diff
@@ -0,0 +1,43 @@
+const fs = require("fs");
+const os = require("os");
+const promptAsync = require("../util/promptAsync");
+
+const hiddenMaestroDirPath = () => {
+  const homedir = os.homedir();
+  const dir = "/.maestro";
+
+  return homedir + dir;
+};
+
+const createHiddenMaestroDir = () => {
+  const hiddenDirPath = hiddenMaestroDirPath();
+
+  if (!fs.existsSync(hiddenDirPath)) {
+    fs.mkdirSync(hiddenDirPath);
+  }
+};
+
+const asyncPromptForAccountInfo = async () => {
+  const accountNum = await promptAsync(
+    "Please enter your AWS Account Number: "
+  );
+  const region = await promptAsync(
+    "Please enter the region for you AWS servises (e.g. us-west-2): "
+  );
```

Comment:
There's a typo: it should be "services", not "servises".

Reply:
Good catch

Reply:
Can be fixed